docs/index.rst

```diff
@@ -488,6 +488,26 @@ Fields can also be of type ``Callable``:
     callable, no validation of arguments, their types or the return
     type is performed.
 
+Color Type
+..........
+
+You can use ``Color`` data type for storing colors as per
```

Comment:
Best to give a bullet point list with a short description of each permitted format